pkg/notifier/config.go

```diff
@@ -23,9 +24,9 @@ func (cfg *NotifierConfig) IsValid() error {
 	switch cfg.ConfigManager.Source {
 	case string(OCM):
 		return nil
+	case string(LOCAL):
+		return nil
```

Comment:
Considering this, the end user for ARO/OCP would always have to specify the source as "LOCAL" in the configmap explicitly. What about considering LOCAL as the default source if no other source (Eg. OCM) is mentioned explicitly?